spec/features/candidates/registrations_spec.rb

```diff
@@ -155,8 +168,12 @@
     expect(page).to have_text "Teaching subject - second choice Mathematics"
     expect(page).to have_text "DBS check document Yes"
 
-    # Submit placement request form successfully
+    # Send email confirmation
     click_button 'Accept and send'
-    expect(page).to have_text 'Your placement request has been sent'
+    expect(page).to have_text \
+      "Click the confirmation link in the email we’ve sent to the following email address to confirm your request for a placement at Test School:\ntest@example.com"
+
+    # Visit email confirmation link
+    # expect(page).to have_text 'Your placement request has been sent'
```

Comment:
Does this need to be removed or uncommented?

Reply:
This is just a placeholder, it'll get commented in in my next pr